detection_rules/schema.py

```diff
@@ -16,6 +16,7 @@
 VERSION_PATTERN = r'\d+\.\d+\.\d+'
 RULE_LEVELS = ['recommended', 'aggressive']
 MATURITY_LEVELS = ['development', 'testing', 'staged', 'production', 'deprecated']
+OPERATORS = ['equals']
```

Comment:
The supported `operators` for Severity/RiskScore mapping is currently only `equals` see [rules schema](https://github.com/elastic/kibana/blob/591e10355aa5e9fe590361fd22483257d146b9ba/x-pack/plugins/security_solution/common/detection_engine/schemas/common/schemas.ts#L209-L212).